pages.de/common/django-admin.md

```diff
@@ -0,0 +1,20 @@
+# django-admin
+
+> Djangos Hilfstool für administrative Aufgaben.
```

Comment:
Hilfstool seems like a really weird neologism to me, I'd prefer Werkzeug or just Tool

```suggestion
> Djangos Werkzeug für administrative Aufgaben.
```